### DISCUSSION TITLE
关于几个产品的疑问

### DISCUSSION THREAD

**Discussion by daoshouchen** [secretflow/secretflow#1076] [Q&A]
1 基于 kuscia 搭建的计算框架和 mvp 之间是什么关系，mvp 包是否包含了 kuscia 框架的所有吗？ 2 我直接使用 mvp 体验包搭建双节点，允许生产计算是否可行？  3 scql 我想集成，安装了是不是和其他的重复了？ scql 底层是不是又安装了一次 kuscia和secretflow ？小白越看越迷糊.

**Comment by xiushuiguande** [comment]
hi，问题在issue中回复
https://github.com/secretflow/secretflow/issues/1077
